weatherApp/ViewController.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  ViewController.swift
+//  weatherApp
+//
+//  Created by Luyanda Sikithi on 2022/11/02.
+//
+
+import UIKit
+import Resolver
+class ViewController: UIViewController {
+
+    @IBOutlet weak var tableView: UITableView!
+    
+    @Injected var viewModel: WeatherViewModel
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupTableView()
+        getWeather()
+    }
+    
+    private func setupTableView() {
+        tableView.dataSource = self
+        tableView.delegate = self
+        
+        tableView.register(UINib(nibName: "CitiesTableViewCell", bundle: nil), forCellReuseIdentifier: "CitiesTableViewCell")
```

Comment:
To avoid the need to maintain strings, which are easily missed when renaming classes you could write this as:

UINib(nibName: String(describing: CitiesTableViewCell.self,